cmd/wallet-web/src/pages/Credentials.vue

```diff
@@ -123,10 +123,6 @@
     <!-- Loading State -->
     <skeleton-loader v-if="loading" type="vault" />
     <!-- Error State -->
-    <span v-else-if="loadingStatus === 'failed'">
-      <b>Warning:</b> Failed to connect to server. Your wallet can not participate in secured
-      communication.
-    </span>
```

Comment:
We should at least console error it if the loading status is failed

Reply:
this should not have been deleted, this was for my local env only. not pushing this change.